lib/gli/command_finder.rb

```diff
@@ -9,16 +9,16 @@ class CommandFinder
 
     def initialize(commands, options = {})
       self.options = DEFAULT_OPTIONS.merge(options)
-      self.commands_with_aliases = expand_with_aliases(commands)
+      @commands_with_aliases = expand_with_aliases(commands)
```

Comment:
Ruby complains about using a private accessor. I don't particularly agree with it too, but at some situations, it might make sense. To avoid the warning, either make the accessor public, or access the instance variable directly.